skimage/measure/sopoly.py

```diff
@@ -0,0 +1,1789 @@
+import numpy as np
+import math
+from functools import reduce
+
+
+def _shift_indices(a1, a2, n_vertices, ex_start=True, ex_end=True):
+    """ Shifts the position of the indices in a way that wraps them connecting
```

Comment:
```suggestion
    """Shift the position of the indices in a way that wraps them connecting
```
cf. https://peps.python.org/pep-0257/#multi-line-docstrings

Reply:
Thank you again @mkcor for reviewing my code. I'll fix all the docstrings following the pep that you referenced.